### PR TITLE
Release notes update with the core-clients contributions

### LINKED ISSUE #32990: Upgrade release-notes with the contributions related to core-clients
### Description

This task is about make sure that issues with this label are documented in the Keycloak 26 release notes: https://github.com/keycloak/keycloak/issues?q=is%3Aissue+label%3Arelease%2F26.0.0+is%3Aclosed+label%3Arelease-notes+label%3Ateam%2Fcore-clients .

It is possible that some of them are already documented there. In this case, we need to make sure to not document them twice. This is to be doublechecked.

### PR DESCRIPTION
**Pull request by mposolda**
closes #32990

### CODE REVIEW

File: docs/documentation/release_notes/topics/26_0_0.adoc [hunk 1]
```diff
@@ -112,10 +112,49 @@ Starting with {project_name} 26, the Organizations feature is fully supported.
 
 Now {project_name} allows configuring ECDH-ES, ECDH-ES+A128KW, ECDH-ES+A192KW or ECDH-ES+A256KW as the encryption key management algorithm for clients. The Key Agreement with Elliptic Curve Diffie-Hellman Ephemeral Static (ECDH-ES) specification introduces three new header parameters for the JWT: `epk`, `apu` and `apv`. Currently {project_name} implementation only manages the compulsory `epk` while the other two (which are optional) are never added to the header. For more information about those algorithms please refer to the link:https://datatracker.ietf.org/doc/html/rfc7518#section-4.6[JSON Web Algorithms (JWA)].
 
+There is also new key provider `ecdh-generated` for generating realm keys and the support for ECDH algorithms is added into Java KeyStore provider.
```
Comment (andymunro) [inline docs/documentation/release_notes/topics/26_0_0.adoc:115]:
```suggestion
Also, a new key provider, `ecdh-generated`, is available to generate realm keys and support for ECDH algorithms is added into the Java KeyStore provider.
```

File: docs/documentation/release_notes/topics/26_0_0.adoc [hunk 2]
```diff
@@ -112,10 +112,49 @@ Starting with {project_name} 26, the Organizations feature is fully supported.
...
+
 ifeval::[{project_community}==true]
 Many thanks to https://github.com/justin-tay[Justin Tay] for the contribution.
 endif::[]
 
+= DPoP improvements
+
+There are improvements in the DPoP (OAuth 2.0 Demonstrating Proof-of-Possession) preview feature. The DPoP is now supported for all grant types.
+Until the previous release, it was supported only for the `authorization_code` grant type. There is also support for the DPoP token type on the UserInfo endpoint.
```
Comment (andymunro) [inline docs/documentation/release_notes/topics/26_0_0.adoc:124]:
```suggestion
The DPoP (OAuth 2.0 Demonstrating Proof-of-Possession) preview feature has improvements. The DPoP is now supported for all grant types.
With previous releases, this feature was supported only for the `authorization_code` grant type. Support also exists for the DPoP token type on the UserInfo endpoint.
```

File: docs/documentation/release_notes/topics/26_0_0.adoc [hunk 3]
```diff
@@ -112,10 +112,49 @@ Starting with {project_name} 26, the Organizations feature is fully supported.
...
+
+ifeval::[{project_community}==true]
+Many thanks to https://github.com/Captain-P-Goldfish[Pascal Knüppel] for the contribution.
+endif::[]
+
+= Client Attribute condition in Client Policies
+
+The condition based on the client-attribute was added into Client Policies. This condition can be used to specify that it is applied for the clients
+with the specified client attribute having specified value. It is possible to use either AND or OR condition when evaluating this condition as mentioned in the documentation
+for client policies.
```
Comment (andymunro) [inline docs/documentation/release_notes/topics/26_0_0.adoc:134]:
```suggestion
The condition based on the client-attribute was added into Client Policies. You can use condition to specify for the clients
with the specified client attribute having specified value. It is possible to use either an AND or OR condition when evaluating this condition as mentioned in the documentation
for client policies.
```

File: docs/documentation/release_notes/topics/26_0_0.adoc [hunk 4]
```diff
@@ -112,10 +112,49 @@ Starting with {project_name} 26, the Organizations feature is fully supported.
...
+
+ifeval::[{project_community}==true]
+Many thanks to https://github.com/y-tabata[Yoshiyuki Tabata] for the contribution.
+endif::[]
+
+ifeval::[{project_community}==true]
+= OpenID for Verifiable Credential Issuance
+
+The OpenID for Verifiable Credential Issuance (OID4VCI) is still experimental feature in {project_name}, but it was greatly improved in this release. Lots of development and discussions
+happen in the https://github.com/keycloak/kc-sig-fapi[Keycloak OAuth SIG]. Anyone from the Keycloak community is welcome to join.
```
Comment (andymunro) [inline docs/documentation/release_notes/topics/26_0_0.adoc:144]:
```suggestion
The OpenID for Verifiable Credential Issuance (OID4VCI) is still an experimental feature in {project_name}, but it was greatly improved in this release. You will find significant development and discussions
in the https://github.com/keycloak/kc-sig-fapi[Keycloak OAuth SIG]. Anyone from the Keycloak community is welcome to join.
```

**Review by andymunro — CHANGES_REQUESTED** [review]
Looks good, @mposolda

**Comment by mposolda** [comment]
@andymunro Thanks!

**Pull request merged**